### PR TITLE
adapt account creation doc

### PR DESCRIPTION
**Pull request by gtema**
This PR is created on base of https://github.com/opentelekomcloud-docs/docs_on_docs/pull/26 to reimplement it on top of current master branch ignoring all not relevant local commits:
```
git checkout -b Nils-Magnus-toplevel_struct main
git pull git@github.com:Nils-Magnus/docs_on_docs.git toplevel_struct
git cherry-pick a1eb128c76e67a69cb3e3195f0831b527ff1bcc2
git checkout -b toplevel_struct
git push
```

### CODE REVIEW

File: doc/source/account creation
Comment (Nils-Magnus) [inline doc/source/account creation:1]: This PR is mainly about re-syncing a few files.

File: doc/source/account creation [hunk 1]
```diff
@@ -1,16 +0,0 @@
...
-2.	On the top right corner is a button with “sign up”
-3.	Enter your corporate Email and press continue
-4.	Create strong and memorable password
-5.	Enter a username e.g.  joedoe
-6.	Would you like to receive updates? I suggest opting out with entering “n”
-7.	Verify your account by solving the task to prove you aren´t a bot
-8.	Hit a “create an account”
-9.	You are almost done! GitHub just sent you a launch you that has to be entered to create the account ultimately. Note: The Code has to be entered manually and can´t be entered via “copy & paste” 
-10.	You are asked with how many people you are working with and if you are a student or a teacher. This personalization can be skipped at the bottom of the Website. I would suggest skipping this step. 
-11.	Congratulations you just created you GitHub account
```
Comment (Nils-Magnus) [inline doc/source/account creation:16]: dropped this file without `rst`extension.

File: doc/source/account_creation.rst [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+
+9.  You are almost done! GitHub just sent you a launch you that has to
+    be entered to create the account ultimately. Note: The code has to
+    be entered manually and you can't enter it via copy & paste.
+
+10. You are asked with how many people you are working with and if you
+    are a student or a teacher. This personalization can be skipped at
+    the bottom of the Website. I suggest skipping this step.
+
+11. Congratulations! You just created your GitHub account!
```
Comment (Nils-Magnus) [inline doc/source/account_creation.rst:37]: ... and fixing the right one with proper formatting.

File: doc/source/index.rst [hunk 3]
```diff
@@ -2,31 +2,33 @@
...
+Therefore, all public facing documentation is maintained in the open,
+by open-source components. All steps of the process are publicly
+visible and transparent. This eases the collaboration of our
+suppliers, OTC staff, and even customers or users. All can propose
+changes, all can participate in the review process, and each team can
+appoint arbitrary persons to approve changes, ensuring the integrity
+of the overall documentation. All changes are recorded and archived,
+so changes can be rolled back, and audit trails are available even for
+the tiniest change.
+
```
Comment (Nils-Magnus) [inline doc/source/index.rst:22]: reformatting

File: doc/source/index.rst [hunk 4]
```diff
@@ -2,31 +2,33 @@
...
 ==============================
 
 The content of the projects is owned by the Open Telekom Cloud Squads
 responsible for the services respectively. Each team can appoint one
-or several Squad members (or external contributors, if
-suitable). Typically, one documentation lead and a deputy are
-recommended, but other setups are also possible.
+or several Squad members (or external contributors, if suitable).
+Typically, one documentation lead and a deputy are recommended, but
+other setups are also possible.
```
Comment (Nils-Magnus) [inline doc/source/index.rst:31]: reformatting

File: doc/source/index.rst [hunk 5]
```diff
@@ -48,6 +50,7 @@ was initiated by the Ecosystem Squad. It was presented to the Product
 Owner Community and during the XXXXXXXXXXX meeting/call/board. It was
```
Comment (Nils-Magnus) [inline doc/source/index.rst:50]: should be updated in a later PR

File: doc/source/index.rst [hunk 6]
```diff
@@ -75,6 +78,7 @@ Content
    change_proposal_process
    reviewer
    approver
+   account_creation
```
Comment (Nils-Magnus) [inline doc/source/index.rst:81]: adding this file to the main page

**Review by Nils-Magnus — APPROVED** [review]
Explains the necessary. Updates formatting. LGTM

**Comment by Nils-Magnus** [comment]
recheck

**Review by tischrei — APPROVED** [review]

**Review by SebastianGode — APPROVED** [review]

**Pull request merged**